backend/springCrudAPI/src/main/java/com/example/springBoot/entity/User.java

```diff
@@ -49,15 +49,16 @@ public User() {
 		
 	}
 		
-	public User(String firstName, String lastName, String email, String Password, String Role, String Bio){
+	public User(long ID, String firstName, String lastName, String email, String Password, String Role, String Bio){
```

Comment:
I am a little worried about changing this type since we have a lot of usage in other services now, and I am not sure how it will affect others. How did you test this PR?